packages/contracts/contracts/Unirep.sol

```diff
@@ -748,4 +748,44 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
         epochTransitionCompensation[msg.sender] = 0;
         Address.sendValue(payable(msg.sender), amount);
     }
+
+    function globalStateTreeDepth() public view returns (uint8) {
```

Comment:
Because we create getters for config values I think we should declare `config` as private data.

Reply:
Ah we still use the `config` variable off chain to load the full config in 1 rpc call. These getters are mostly useful on chain because the config struct is returned as a list instead of a struct in other contracts.

Reply:
Thanks <3